langchain_tools_example.py

```diff
@@ -27,9 +38,7 @@ def parse_drafts(output, query):
     return output
 google_drafts_tool = GoogleGetDraftsTool(client=client, user_id=user_id, llm=input_llm, parse_output=parse_drafts)
 agent = initialize_agent([google_drafts_tool], agent_llm, agent='chat-zero-shot-react-description', verbose=True)
-result = agent.run("What are my Google drafts?")
-
-exit()
+result = agent.run("What are my Google drafts with q=null and pageToken='1'?")
```

Comment:
We've tested without explicit q values right?

Reply:
Yep, haven't fully tested but have tested with q='to:tevon@bruinen.co' and it works great